AppCenterDistribute/AppCenterDistribute/MSACDistributeUtil.m

```diff
@@ -132,4 +135,23 @@ + (BOOL)isValidUpdateTrack:(MSACUpdateTrack)updateTrack {
   return updateTrack == MSACUpdateTrackPublic || updateTrack == MSACUpdateTrackPrivate;
 }
 
++ (ASPresentationAnchor)getPresentationAnchor API_AVAILABLE(ios(13)) {
+  UIApplication *application = MSAC_DISPATCH_SELECTOR((UIApplication * (*)(id, SEL)), [UIApplication class], sharedApplication);
+  NSSet *scenes = MSAC_DISPATCH_SELECTOR((NSSet * (*)(id, SEL)), application, connectedScenes);
+  NSObject *windowScene;
+  for (NSObject *scene in scenes) {
+    NSInteger activationState = MSAC_DISPATCH_SELECTOR((NSInteger(*)(id, SEL)), scene, activationState);
+    if (activationState == 0 /* UISceneActivationStateForegroundActive */) {
```

Comment:
let's move the comment above this condition

Reply:
It was copied from another palce. I think it looks better here because the comment is to `0` value.

Reply:
I think we should remove these `MSAC_DISPATCH_SELECTOR` and do calls normally, but we need to check if it will not break anything. It was introduced for compatibility with older Xcode versions which we don't support anymore I believe.